p4-16/psa/PSA.mdk

```diff
@@ -1288,45 +1288,203 @@ to generate values.
 [INCLUDE=psa.p4:Random_extern]
 ```
 
+
+## Packet Generation
+
+\TODO: is generating a new packet and sending it to the stream or is
+it adding a header to the current packet and sending it to the
+stream (copying or redirecting).
+
+```
+[INCLUDE=psa.p4:Digest_extern]
+```
+
+
```

Comment:
@samar-abdi Sorry, one more nit to pick -- the group of lines added above looks unrelated to the rest of this.  Not sure if it is easy to remove from your PR?